ui.apps/src/main/content/jcr_root/apps/aemtraining/components/content/footer/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
+          xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="News Card Header"
+          sling:resourceType="cq/gui/components/authoring/dialog"
+          extraClientlibs="[aemtraining.components.newscard.editor]">
```

Comment:
You know you add extraClientlibs property when you actually need it (it contains the ID of a client lib you use to add some more javascript functionslity ro CSS to the dialog itself). 
If you haven't got a clienlib with such ID elsewhrere in your project, you don't need the `extraClientlibs` property